lerna.json

```diff
@@ -0,0 +1,10 @@
+{
+	"command": {
+		"publish": {
+			"message": "chore(release): publish"
+		}
+	},
+	"ignoreChanges": [ "**/CHANGELOG.md", "**/test/**" ],
```

Comment:
As I understand it, the only thing `lerna` reads for packages are `package.json` files, so nothing here fits that criteria?

Reply:
That config comes from WC Admin. I believe excluding the changelog allows you to fix typos, etc. in the change log without bumping the version. Even though the `e2e-environment` won't likely have tests, some of the components we create might (eg. https://github.com/woocommerce/woocommerce-admin/tree/main/packages/number/src).